core_fw/ipc_radio/overlay-bt_hci_ipc.conf

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_MBOX=y
+CONFIG_IPC_SERVICE=y
+CONFIG_IPC_SERVICE_BACKEND_RPMSG=y
```

Comment:
I thought the ipc_service backend was selected using the device tree. What if the device tree selected icmsg while this overlay enables rpmsg?

Reply:
Good catch, removed the line